test/libsolidity/smtCheckerTests/verification_target/target_in_constructor_of_same_source_contract_not_analyzed_when_not_selected_and_not_called_from_selection.sol

```diff
@@ -0,0 +1,13 @@
+contract B {
+   constructor() { fail(); }
+
+   function fail() pure internal { assert(false); }
```

Comment:
```suggestion
    constructor() { fail(); }

    function fail() pure internal { assert(false); }
```

`target_in_same_source_contract_not_analyzed_when_not_selected_and_not_called_from_selection.sol` has that too.

Reply:
Fixed.